elitech/__init__.py

```diff
@@ -40,7 +40,7 @@
 class Device:
     def __init__(self, serial_port):
         if serial_port is not None:
-            self._ser = serial.Serial(serial_port, 115200, timeout=3)
+            self._ser = serial.Serial(serial_port, baudrate=115000, timeout=3)
```

Comment:
I think it is same code. Second parameter is baudrate. 
https://pythonhosted.org/pyserial/pyserial_api.html#classes
You explicitly wrote it.